docs/contributing/contrib.md

```diff
@@ -1,19 +1,36 @@
 ## PaSh on Docker: A Pocket Guide
 
-This is a pocket guide for running PaSh in a docker container.
+This is a pocket guide for running PaSh in a Docker container.
 
 ### Loading image
 
-The following will pull and launch the latest image (tag: `pash:latest`) using the name `pash-play`
+#### From Docker Hub
+
+The following will pull the latest Docker image (tag: `pash:latest`),
+and start a new container using the name `pash-play`.
 
 ```sh
 docker pull binpash/pash
 docker run --name pash-play -it pash
 ```
+
 This will use `pash-play` for the container (to be able to start/stop it later) and `-it` runs it interactively (subsuming the next command).
 
 To restart after you exit, run `docker start -i pash-play` (flag `-i` starts it interactively.)
 
+
+#### From source code
```

Comment:
Is this something you added and related to this PR?

Reply:
It was related when I kept the other documentation change, since it created redundancies with this file. I'll revert this.